examples/webpack2/src/main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Button, Container, Header } from 'semantic-ui-react'
```

Comment:
This example contains bundle analyzer, so you can see same picture as [there](https://github.com/Semantic-Org/Semantic-UI-React/issues/524#issuecomment-289219955). It loads build from `es` dir.